### ISSUE TITLE
Update automated installation instructions

### ISSUE THREAD

**Issue by nicholas-fedor** [AdguardTeam/AdGuardHome#7261]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

Piping the output stream to shell execution `curl -s -S -L https://raw.githubusercontent.com/AdguardTeam/AdGuardHome/master/scripts/install.sh | sh -s -- -v` introduces a potential security vulnerability and/or unintended behavior when experiencing network issues.

Best practice is to save the file first prior to execution.

### Proposed solution

tl;dr - save the output file to disk prior to execution.

Example: `curl -o install-adguardhome.sh https://raw.githubusercontent.com/AdguardTeam/AdGuardHome/master/scripts/install.sh && sh ./install-adguardhome.sh -v`

[Example of implemented fix](https://github.com/nicholas-fedor/AdGuardHome#automated-install-linuxunixmacosfreebsdopenbsd)
